ethcore/src/client/client.rs

```diff
@@ -144,32 +144,70 @@ impl SleepState {
 /// Blockchain database client backed by a persistent database. Owns and manages a blockchain and a block queue.
 /// Call `import_block()` to import a block asynchronously; `flush_queue()` flushes the queue.
 pub struct Client {
+	/// Flag used to disable the client forever. Not to be confused with `liveness`.
+	///
+	/// For example, auto-updater will disable client forever if there is a
+	/// hard fork registered on-chain that we don't have capability for.
+	/// When hard fork block rolls around, the client (if `update` is false)
+	/// knows it can't proceed further.
```

Comment:
the fields here are not public. comments are great, but they should be `//`

Reply:
I see no reason to enforce using `//`. In multiple places in our codebase we use `//` for private properties and structs.

Reply:
RLS provides hover hints for fields commented with `///` only.